package.json

```diff
@@ -26,6 +26,7 @@
 	},
 	"dependencies": {
 		"await-to-js": "^3.0.0",
+		"axios": "^1.3.5",
```

Comment:
Can we use `fetch` instead of adding a new package?

Reply:
It's an unused dependency at this point. So I have removed it.